### ISSUE TITLE
Will speaker note support markdown syntax?

### ISSUE THREAD

**Issue by fntsrlike** [hiroppy/fusuma#151]
## Info

- Operating System: macOS Mojave
- Node Version: node@10
- Browser Name and Version: Safari 12.1.1

## Type

- [ ] bug
- [X] feature request

### What is the motivation or use-case for the feature?
I love fusuma. I can do my slides by writing markdown document.
I expect there also support markdown on speaker note, but it doesn't.

I want to put the reference link in speaker note as following.

```md
[Fusuma](https://github.com/hiroppy/fusuma/)
```

but it just display with plain text.

I am expected that will speaker note support markdown syntax?

**Comment by hiroppy** [comment]
Yep, currently, fusuma doesn't support Markdown. Do you have use-case? Because the speaker note is watched by only a speaker so I thought I don't need to support Markdown to the speaker note.(e.g. clickable links)

**Comment by vivaxy** [comment]
Vote for the feature.
Reasons:
- Write lists, tables, syntax highlight code in markdown, which provides a better readability.
- Use images in notes.
- Writing markdown throughout `fusuma` brings good experience.

**Comment by kospl** [comment]
Is this still an issue? We're looking for Markdown to presentation generation tool and this tool looks like checking a lot of boxes.
